circle.geojson

```diff
@@ -0,0 +1,5705 @@
+{
```

Comment:
It just bloats the git repo to check this in; how about we add it in `.gitignore` instead?